ast/src/circuits/circuit_member.rs

```diff
@@ -15,7 +15,6 @@
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
 use crate::{Expression, Function, Identifier, Type};
```

Comment:
Not relevant to this PR, but I feel we should discuss a style standard for this/enable it in Clippy. I always put newlines between internal imports, built-in imports, and external imports for readability and clarity. Just a discussion to have so we are consistent in the future.

Reply:
The standard in rustc today seems to be to separate like you say: https://doc.rust-lang.org/nightly/nightly-rustc/src/rustc_parse/parser/item.rs.html

My preference however would be to be "imports are not important; I can throw an import into the list anywhere and rustfmt will take care of it."

Reply:
Not against that, but it makes it less readable/harder to follow, which I personally believe is important for an open-source code base. I'll make an issue from this, and we can discuss it with the group at a later point ^^.

Reply:
Hmm... Feels like a small enough issue that I wouldn't want to waste the team's time on that discussion. I can revert this change. :)